<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-28T18:48:28 IST" name="Avast Security Suite" finished-at="2025-08-28T18:48:44 IST" duration-ms="15835">
    <groups>
    </groups>
    <test started-at="2025-08-28T18:48:28 IST" name="Avast Tests" finished-at="2025-08-28T18:48:44 IST" duration-ms="15835">
      <class name="com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning">
        <test-method is-config="true" signature="setupReport()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:28 IST" name="setupReport" finished-at="2025-08-28T18:48:28 IST" duration-ms="147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:28 IST" name="setup" finished-at="2025-08-28T18:48:42 IST" duration-ms="13886" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="setupPages()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:42 IST" name="setupPages" finished-at="2025-08-28T18:48:42 IST" duration-ms="63" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupPages -->
        <test-method signature="testToastMessageAfterScan()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:42 IST" name="testToastMessageAfterScan" finished-at="2025-08-28T18:48:43 IST" duration-ms="1036" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.fail(String)" because "com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning.test" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.fail(String)" because "com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning.test" is null
at com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning.testToastMessageAfterScan(Validating_The_Toast_Message_After_Scanning.java:78)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToastMessageAfterScan -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:43 IST" name="tearDown" finished-at="2025-08-28T18:48:44 IST" duration-ms="793" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDownReport()[pri:0, instance:com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning@196a42c3]" started-at="2025-08-28T18:48:44 IST" name="tearDownReport" finished-at="2025-08-28T18:48:45 IST" duration-ms="790" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownReport -->
      </class> <!-- com.avastsecurity.Test.Validating_The_Toast_Message_After_Scanning -->
    </test> <!-- Avast Tests -->
  </suite> <!-- Avast Security Suite -->
</testng-results>
